src/components/common/FileUploadPreview.vue

```diff
@@ -24,27 +24,33 @@
 </template>
 
 <script lang="ts">
-import { Component, Emit, Prop, Watch } from 'vue-property-decorator'
-import Vue from 'vue'
+// Libraries
+import { Component, Emit, Mixins, Prop, Watch } from 'vue-property-decorator'
+
+// Mixins
+import { DocumentMixin } from '@/mixins'
+import { PdfPageSize } from '@/enums'
 
 @Component({
   components: {}
 })
-export default class FileUploadPreview extends Vue {
+export default class FileUploadPreview extends Mixins(DocumentMixin) {
   @Prop() inputFile: File
   @Prop({ default: 'File' }) inputFileLabel: string
   @Prop({ default: true }) isRequired: boolean
   @Prop({ default: 0 }) maxSize: number // in KB
+  @Prop({ default: null }) pdfPageSize: PdfPageSize
   @Prop({ default: false }) showErrors!: boolean
   @Prop({ default: '' }) customErrorMessage: string
 
-  private fileUpload = null
-  private customErrorMessages = []
-
+  // Refs
   $refs: {
     fileUploadInput: HTMLFormElement
   }
 
+  private fileUpload = null
+  private customErrorMessages = []
```

Comment:
Array of what type?

Reply:
added type to declaration.  i.e ` private customErrorMessages:string[] = []`